manifest-beta.json

```diff
@@ -1,7 +1,7 @@
 {
 	"id": "shiki-highlighter",
 	"name": "Shiki Highlighter",
-	"version": "0.3.2",
+	"version": "0.3.3",
```

Comment:
Please don't increment the version number. This is done by the release script automatically.